core/linux-dist/x11.cpp

```diff
@@ -91,14 +91,32 @@ void input_x11_handle()
 						}
 					}
 #endif
-					else if (e.type == KeyRelease && e.xkey.keycode == 95) // F11 button
+					else if (e.type == KeyRelease && e.xkey.keycode == KEY_F11)
 					{
 						x11_fullscreen = !x11_fullscreen;
 						x11_window_set_fullscreen(x11_fullscreen);
 					}
 					else
 					{
 						int dc_key = x11_keymap[e.xkey.keycode];
+
```

Comment:
I think these should go though the `x11_keymap` lookup. Dreamcast itself uses just 16 bits, so we can use special values for the triggers.

Reply:
I'm sorry, but I don't understand what you mean here.

Reply:
Ah, I think I get it.
Fixing the sliders below (``TODO: Fix sliders``) and use 
````c
if (dc_key == DC_BTN_DPAD_DOWN)
````
instead of
````c
if (e.xkey.keycode == KEY_F)
````

Reply:
Yup!

Reply:
Ok, then this should be done.